pipeline/beam_tables.py

```diff
@@ -397,7 +415,17 @@ def _data_to_load(self,
         'gs://firehook-scans/echo/CP_Quack-echo-2020-08-23-06-01-02/results.json']
     """
     if incremental_load:
-      if table_name:
+      if table_name and gcs_folder:
+        full_table_name = self._get_full_table_name(table_name)
+        table_existing_sources = _get_existing_bq_datasources(
+            full_table_name, self.project)
+        gcs_existing_sources = _get_existing_gcs_datasources(
+            gcs_folder, self.project)
+        table_set = set(table_existing_sources)
+        gcs_set = set(gcs_existing_sources)
+        self._compare_sets(table_set, gcs_set)
```

Comment:
only check this when planning to write to both datasources

Reply:
It does already, since it only enters this branch when table_name and gcs_folder have been set.